07-AzTS REST APIs/README.md

```diff
@@ -9,17 +9,17 @@
 
 The Azure Tenant Security Solution (AzTS) provides REST APIs which can be leveraged to scan subscription(s) and get control scan result for subscription(s). This is alternative for AzTS UI to get insights about security compliance from AzTS perspective. 
 
-## Set up (For AzTS Admin Only)
+## Set-up (For AzTS Admin Only)
```

Comment:
Shouldn't it be "setup" instead of "set up" OR "set-up" (saw the word setup being used in other places in the repo. Hence the ask on aligning)